src/explorer/weights/WeightSlider.test.js

```diff
@@ -3,7 +3,7 @@
 import React from "react";
 import {shallow} from "enzyme";
 
-import {WeightSlider, formatWeight} from "./WeightSlider";
+import {WeightSlider, formatWeight, WEIGHT_SLIDER_MIN} from "./WeightSlider";
```

Comment:
Usually, when we have tests converting among two representations, we
like to see “roundtrip tests”: that converting in one direction and then
the other gets you the original value. In this case, this isn’t exactly
what we want, because one of the directions is _supposed_ to be lossy.
But converting from slider-value to weight-value and back to
slider-value should always be the identity operation. It would be nice
to test that on some interesting values.

Reply:
Added round-trip tests.